modules/porous_flow/doc/content/modules/porous_flow/hysteresis.md

```diff
@@ -310,3 +310,48 @@ The remainder of the input file is standard, with the inclusion of the hystereti
 The result is [hys_1phase_3_fig].
 
 !media media/porous_flow/hys_1phase_3.png caption=The result of a single-phase simulation in which an external pump removes and adds water to a porous material in order to observe the hysteretic capillary pressure.  id=hys_1phase_3_fig
+
+### Two-phase example using the PP formulation
+
+A simulation that simply adds gas then removes gas from a 2-phase system to observe the hysteretic capillary pressure is explored in this section.  The gas flux is controlled by the Postprocessor
+
+!listing modules/porous_flow/test/tests/hysteresis/2phasePP.i start=[./flux] end=[./hys_order]
+
+and the DiracKernel
+
+!listing modules/porous_flow/test/tests/hysteresis/2phasePP.i block=DiracKernels
+
+The remainder of the input file is standard, with the inclusion of the hysteretic capillary pressure:
+
+!listing modules/porous_flow/test/tests/hysteresis/2phasePP.i start=[./hys_order_material] end=[]
+
+The result is [hys_2phasePP_1_fig].
+
+!media media/porous_flow/hys_2phasePP_1.png caption=The result of a two-phase simulation using a PP formulation in which an external pump adds to and removes gas from a porous material in order to observe the hysteretic capillary pressure.  id=hys_2phasePP_1_fig
+
+### Two-phase example using the PS formulation
+
+A simulation that simply adds gas, then removes gas, and adds it again to a 2-phase system to observe the hysteretic capillary pressure is explored in this section.  The gas flux is controlled by the Postprocessor
+
+!listing modules/porous_flow/test/tests/hysteresis/2phasePS_2.i start=[./flux] end=[./hys_order]
+
+and the DiracKernel
+
+!listing modules/porous_flow/test/tests/hysteresis/2phasePS_2.i block=DiracKernels
+
+The remainder of the input file is standard, with the inclusion of the hysteretic capillary pressure:
+
+!listing modules/porous_flow/test/tests/hysteresis/2phasePS_2.i start=[./hys_order_material] end=[]
+
+The result is [hys_2phasePS_2_fig].
+
+!media media/porous_flow/hys_2phasePS_2.png caption=The result of a two-phase simulation using a PS formulation in which an external pump adds, removes and then adds gas to a porous material in order to observe the hysteretic capillary pressure.  id=hys_2phasePS_2_fig
+
+
+## Hysteretic capillary-pressure implementation remarks
```

Comment:
This documentation is really great, except that the following repeats from a warning earlier on this page. Not sure if you mean to remove the warning, or perhaps shorten the warning to refer the readers to this section?

Reply:
removed the warning and re-worded its paragraph